modules/dynamic-plugin-localization.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * web_console/dynamic-plugin/overview-dynamic-plugin.adoc
+
+:_mod-docs-content-type: Procedure
+[id="dynamic-plugin-api_{context}"]
+= Translating messages with react-i18next
+
+The link:https://github.com/openshift/console-plugin-template[plugin template] demonstrates how you can translate messages with link:https://www.i18next.com/[react-i18next]. The `v1alpha1` `apiVersion` of `ConsolePlugin` supports `console.openshift.io/use-i18n` annotation.
+
+.Prerequisites
+* You must have the plugin template cloned locally.
+* You must be running the {product-title} web console in a container. You can use either Docker or Podman 3.2.0+.
+* You must have `oc` installed.
+
+.Procedure
+
+. Indicate the `ConsolePlugin` contains localization resources. If the annotation is set to `true`, the localization resources from the i18n namespace named after the dynamic plugin are preloaded. If the annotation is set to any other value or is missing on the `ConsolePlugin` resource, localization resources are not preloaded.
++
+[source,terminal]
+----
+spec:
+  backend:
+    service:
+      basePath: /
+      name: console-demo-plugin
+      namespace: console-demo-plugin
+      port: 9001
+    type: Service
+  displayName: OpenShift Console Demo Plugin
+  i18n:
+    loadType: Preload
+  proxy:
+
+----
+
+. Prefix the name with `plugin__` to avoid any naming conflicts. The plugin template uses the `plugin__console-plugin-template` namespace by default, and you must update when you rename your plugin for example, `plugin__my-plugin`.  You can use the `useTranslation` hook , for example:
```

Comment:
```suggestion
. Prefix the name with `plugin__` to avoid any naming conflicts. The plugin template uses the `plugin__console-plugin-template` namespace by default, and you must update when you rename your plugin for example, `plugin__my-plugin`.  You can use the `useTranslation` hook, for example:
```